docs/status.md

```diff
@@ -88,7 +88,7 @@ one of the following tracking labels.
 | dynamic_update_slice     | no            | revisit      | no             | yes             | no          |
 | einsum                   | no            | revisit      | no             | no              | no          |
 | exponential              | yes           | yes          | yes            | yes             | no          |
-| exponential_minus_one    | no            | yes*         | yes*           | yes             | no          |
+| exponential_minus_one    | yes           | yes          | yes            | yes             | no          |
```

Comment:
Did you check that integer operand gives compilation errors and not unimplemented error during runtime?

Reply:
Thanks for pointing it out. Filed an issue #509